Riot/Modules/Room/Views/InputToolbar/RoomInputToolbarView.swift

```diff
@@ -20,8 +20,9 @@ import GBDeviceInfo
 
 extension RoomInputToolbarView {
     open override func sendCurrentMessage() {
-        // Triggers auto-correct if needed.
-        if self.isFirstResponder {
+        // Triggers auto-correct if needed and if it is not a command.
```

Comment:
I would add a comment that this modification should be removed when the Element PR is merged and mainlined to Tchap https://github.com/vector-im/element-ios/pull/7462